Sources/StreamChatUI/ChatChannelList/ChatChannelListVC.swift

```diff
@@ -231,6 +258,27 @@ open class ChatChannelListVC: _ViewController,
             self?.isPaginatingChannels = false
         }
     }
+    
+    /// Shows the error view.
+    open func showErrorView() {
+        channelListErrorView.isHidden = false
+        
+        UIView.animate(withDuration: 0.5) {
+            self.channelListErrorView.center = .init(x: self.channelListErrorView.center.x, y: self.channelListErrorView.center.y - self.channelListErrorViewHeight)
```

Comment:
Indentation should be improved here and in the code below. Have you run the bootstrap script? I also see some formatting errors in the checks.

Reply:
Was able to fix the python issue I was having mentioned above so those issues should be solve in following commits.